soh/assets/soh_assets.h

```diff
@@ -80,3 +80,13 @@ static const ALIGN_ASSET(2) char gFileSelRANDButtonTex[] = dgFileSelRANDButtonTe
 
 #define dgEmptyTexture "__OTR__textures/virtual/gEmptyTexture"
 static const ALIGN_ASSET(2) char gEmptyTexture[] = dgEmptyTexture;
+
+// Tunic Models
+#define dgLinkKokiriSkel "__OTR__objects/object_link_kmdl/gLinkKokiriSkel"
+static const ALIGN_ASSET(2) char gLinkKokiriSkel[] = dgLinkKokiriSkel;
+
+#define dgLinkGoronSkel "__OTR__objects/object_link_gmdl/gLinkGoronSkel"
+static const ALIGN_ASSET(2) char gLinkGoronSkel[] = dgLinkGoronSkel;
+
+#define dgLinkZoraSkel "__OTR__objects/object_link_zmdl/gLinkZoraSkel"
+static const ALIGN_ASSET(2) char gLinkZoraSkel[] = dgLinkZoraSkel;
```

Comment:
I wanted to bring up a minor detail about the naming convention for the path involving kmdl, gmdl, and zmdl. I understand the rationale behind quickly grasping their purpose through either checking the path's end or their linkage. However, personally, I believe that maintaining a non-abbreviated format in the path could enhance clarity. My suggestion would be to opt for names such as "object_link_kokiri_model," "object_link_goron_model," and "object_link_zora_model."

If we're open to further improvement, we might consider enhancing clarity even more by reintroducing the word "Tunic." For instance, using names like "object_link_kokiri_tunic_model" and "dgLinkKokiriTunicSkel." While this might lengthen the names, it could prevent potential misinterpretations and align better with their actual purpose. Just a thought!

Reply:
oh, I can change the names of the skeletons, that actually makes more sense. As for the folders, those were made to match the Twilight Princess equivalents as an homage of sorts, though if you don't like that, I can change it.

Reply:
It's up to you but I prefer clarity over homage, especially if it's not related to the game by itself ^-^